providers/universal-provider/src/types/misc.ts

```diff
@@ -65,6 +65,7 @@ export interface RequestParams {
   topic: string;
   request: RequestArguments;
   chainId: string;
+  id?: number;
```

Comment:
```suggestion
  id: number;
```
`Id` would be always present now so no need to be optional